worker/src/email/resources/sql/enqueue-messages-email.sql

```diff
@@ -7,20 +7,23 @@ BEGIN
 	RETURNING campaign_id INTO selected_campaign_id;
 
 	WITH messages AS 
-	(UPDATE email_messages m SET dequeued_at = clock_timestamp(), updated_at = clock_timestamp() 
-	WHERE m.campaign_id = selected_campaign_id
-	-- enqueue only those that have not been enqueued - this means that when we retry, we will have to set dequeued_at to null
-	AND m.dequeued_at IS NULL
-	-- check for message_id is null because we dont want to enqueue messages that have already been sent
-	AND m.message_id is NULL
-	RETURNING 
-	id,
-	campaign_id, 
-	recipient, 
-	params, 
-	dequeued_at,
-	created_at,
-	updated_at )
+  (
+    UPDATE email_messages m 
```

Comment:
Why didn’t you set received at to null as well?
I think all status related columns should be set to null if we retrying